config/locales/en.yml

```diff
@@ -3381,6 +3381,7 @@ en:
     formats:
       full_js: "D.M.YYYY HH:mm"
       full: "%{date_format} %H:%M"
+      default: "%Y/%m/%d %H:%M"
```

Comment:
is this in any way affected by user date format settings?

Reply:
we used :full format everywhere